docs/blog/20230904-Daeraxa-SeptemberUpdate.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-09-04
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+I want you to act as a Pulsar team member writing a blog post. I will provide titles of recent changes. You will elaborate on each topic and make each paragraph coherent from my ramblings. You will welcome all readers to the September Pulsar community update.
+
+<!-- more -->
+
+# Welcome to the September Community Update!
+
+Welcome to this month's Pulsar community update! In store for you this month we have some massive changes to our CI process, some good news for Windows Chocolatey users, a new option for Pulsar's title bar, some improvements to our ppm `unpublish` command and work on a brand new utility to help clean up elements of a Pulsar installation.
+
+I realise that this probably raises more questions than it answers, so read on to find out more!
+
+## Upheaving our CI setup
+
+<img src="./assets/moving.png" height="200" />
+
+Ever since we started providing downloadable binaries shortly after the Pulsar project properly started, we have been using Cirrus CI as our continuous integration platform. This has worked very well for us for quite some time now, but unfortunately for us, they recently decided to change their free tier and our setup puts us well beyond that free tier and into some quite serious money. At this point we had a couple of choices: either we stick with Cirrus CI by using our donors' money, or we move to another platform. We chose the latter option as we simply could not justify nor sustain the costs.
+
+We did a lot of searching for alternative platforms and ultimately settled on using GitHub actions for everything. This isn't perfect; for example, we cannot use it for our Apple silicon (M1, M2) builds, nor for ARM Linux, for which we are still using Cirrus CI. Over the next week there was a huge amount of effort put in by a number of Pulsar team members, but particularly [@confused-techie], [@DeeDeeG] and [@Meadowsys] to get everything moved and migrated in order to keep our builds building and our rolling releases rolling.
```

Comment:
```suggestion
We did a lot of searching for alternative platforms and ultimately settled on using GitHub actions for nearly everything. This isn't perfect; for example, we cannot use it for our Apple silicon (M1, M2) builds, nor for ARM Linux, for which we are still using Cirrus CI. Over the next week there was a huge amount of effort put in by a number of Pulsar team members, but particularly [@confused-techie], [@DeeDeeG] and [@Meadowsys] to get everything moved and migrated in order to keep our builds building and our rolling releases rolling.
```

I know you touch on this in the next sentence, but feels a bit more clear.

Reply:
yeah good point there